config/zones.json

```diff
@@ -525,8 +527,8 @@
       "coal": 46336,
       "gas": 29731,
       "geothermal": 39,
-      "hydro": 3945,
-      "hydro storage": 6357,
+      "hydro": 4671,
```

Comment:
How are we sure that these capacity are closer to the truth than what was there initially? (readme quotes a federal agency figure, but i haven't looked in detail).

Reply:
For Germany, readme quotes Bundesnetzagentur (BNetzA), which actually should have all the data.
I downloaded the list of power stations [(Kraftwerksliste)](https://www.bundesnetzagentur.de/SharedDocs/Downloads/DE/Sachgebiete/Energie/Unternehmen_Institutionen/Versorgungssicherheit/Erzeugungskapazitaeten/Kraftwerksliste/Kraftwerksliste_2017_2.xlsx) and checked.
In one cell it says "Hinweis: In das deutsche Netz einspeisende Kraftwerksleistungen in Luxemburg, Schweiz und Österreich sind ebenfalls aufgeführt."
"Please note: Generating capacity feeding into the German grid from LUX, CH and AT are listed here as well."

Some of the TSO's area (I think Amprion) expands a bit over to Austria and Luxemburg.
Pumped Storage "Vianden" is located in Luxembourg right next to the border of Germany, but its production goes entirely to Germany. It's a part of the German market, connected to the German grid only and is also reported there. So are some of the generators located in western Austria.

6357 MW is the capacity of the Pumped Storages located in Germany exclusively according to BNetzA, without LUX and AT. If they are a part of the market and deliver to Germany, the proper number from BNetzA is 9440 MW. Entso-e says 8733 MW. Don't know if unavailabilities due to maintenance are included in ENTSO-E (would be more precise) and BNetzA reports entire installed capacities instead.
That's also why general "hydro" is higher.
Seems like the data from BNetzA was not properly/not entirely used. Checking if data is excluded on entso-e (so it will not double and appear in AT or LUX or CH) could be a way to find out. I'm sure at least for LUX it is not included in their installed capacities on entso-e.

Reply:
🔍  Awsome job 🔍  Please add a condensed version of these details in "_comments" so we don't loose track of it for later on!